modules/jenkins/widget.go

```diff
@@ -120,7 +120,12 @@ func (widget *Widget) rowColor(idx int) string {
 func (widget *Widget) jobColor(job *Job) string {
 	switch job.Color {
 	case "blue":
-		return "blue"
+		// Override color if greenBalls boolean param provided in config
+		if wtf.Config.UString("wtf.mods.jenkins.greenBalls") == "true" {
+			return "green"
+		} else {
+			return "blue"
+		}
```

Comment:
What do you think about extending this out to support whatever coloured balls the user wants, defaulting to blue?

Something like this in the config YAML:

```
    wtf.mods.jenkins.ballColor: "green"
```

and in code:

```
    wtf.Config.UString("wtf.mods.jenkins.successBallColor", "blue")
````

Reply:
I'm also assuming those balls indicate success. I don't use Jenkins so if they're indicating something else, the name would reflect that.

Reply:
Yep you're exactly right, the balls indicate the status of the job. By default Jenkins uses blue (don't ask me why). I've knocked this up but if this isn't best practice or there's a better way of doing this you might have to hold my hand a little here!
```
case "blue":
		// Override color if greenBalls boolean param provided in config
		ballColor := wtf.Config.UString("wtf.mods.jenkins.successBallColor", "blue")
		if ballColor != "blue" {
			return ballColor
		} else {
			return "blue"
		}
```

Reply:
Looks good to me